### ISSUE TITLE
Dub package 2 years out of date

### ISSUE THREAD

**Issue by HuskyNator** [dlang/dub#2070]
### System information
Not Applicable

### Bug Description
The dub package hosted by chocolatey seems 2 years out of date.
Google does show a version 1.23.0 exists, however searching using chocolatey or even using PxlBuzzard's profile shows only the 1.11.0 version (from 24 Oct 2018).
This is also the version chocolatey installs.

### How to reproduce?
Simply search for dub on chocolatey's website, which yields: https://chocolatey.org/packages/dub
Or use `choco search dub` / `choco install dub` with chocolatey installed.

### Expected Behavior
I excpected the latest stable version to be given. Namely `1.23.0`.

### Logs
Not Applicable

**Comment by rikkimax** [comment]
Both LDC and DMD distribute dub on Windows. There is no reason to install it separately unless you are developing it.

**Comment by Geod24** [comment]
> Both LDC and DMD distribute dub on Windows. There is no reason to install it separately unless you are developing it.

I disagree. Having it available in various package managers provides more visibility.

CC @PxlBuzzard : Are you still interested in maintaining / updating it ?

**Comment by HuskyNator** [comment]
Additionally, I installed dmd via chocolatey, and believe that requires me to install dub seperately as well.

**Comment by HuskyNator** [comment]
Seems my installing of dub, asides from dmd, using chocolatey, lead to me accidentally using the outdated version, though I could have sworn it did not register as a command before.  . . probably my mistake.

Also, no matter my need for a dub package, I'd say it's ill advised for your readme to state falsehoods.

**Comment by Imperatorn** [comment]
@PxlBuzzard Would you mind updating chocolatey? Pretty please?

**Comment by PxlBuzzard** [comment]
Hi all! Sorry for the delay here, I didn't quite realize that the last year of updates have all been blocked. I have made a few tweaks to the installer based on their automated reviews and hopefully it will pass. The version on chocolatey will be 1.23.0.1 as I already pushed 1.23.0 and I can't update a package using the same version number.

![image](https://user-images.githubusercontent.com/512416/115766490-e63ce000-a35c-11eb-93d3-a46f32a7e17e.png)
